src/tests/order.spec.js

```diff
@@ -0,0 +1,149 @@
+import { expect, test } from "@playwright/test";
+import {
+  adresa,
+  casKonce,
+  casZacatku,
+  contactPerson,
+  email,
+  endDate1,
+  endDate2,
+  ico,
+  number,
+  odberatel,
+  pocet,
+  pocetDospelych,
+  reditelka,
+  startDate1,
+  startDate2,
+  vek,
+} from "../fixtures.orderpage/fixtures";
+import { OrderPage } from "./pages.2homework/order.page";
+
+test.describe("Testing having the posibility to reach the order form for schools", async () => {
+  test.beforeEach(async ({ page }) => {
+    const orderPage = new OrderPage(page);
+    await orderPage.openHomePage();
+  });
+
+  test("Aplikace umožňuje uživateli v menu Pro učitele vytvoření nové Objednávky pro MŠ/ZŠ", async ({
+    page,
+  }) => {
+    const orderPage = new OrderPage(page);
+    await expect(orderPage.forTeacherButton).toBeVisible();
+
+    await orderPage.forTeacherButton.click();
+
+    await expect(orderPage.newOrderButton).toBeVisible();
+  });
+
+  test("Po kliknutí na Pro učitele > Objednávka pro MŠ/ZŠ se otevře formulář, kde může uživatel vyplnit detail objednávky", async ({
+    page,
+  }) => {
+    const orderPage = new OrderPage(page);
+    await orderPage.forTeacherButton.click();
+    await orderPage.newOrderButton.click();
+
+    await expect(orderPage.newOrderPageHeader).toBeVisible();
+  });
+});
+
+test.describe("Filling the order form", async () => {
+  test.beforeEach(async ({ page }) => {
+    const orderPage = new OrderPage(page);
+    await orderPage.openOrderPage();
+  });
+
+  test("Pokud vyplněné IČO neodpovídá žádnému záznamu v ARESu, vyhodí systém chybovou hlášku", async ({
+    page,
+  }) => {
+    const orderPage = new OrderPage(page);
+
+    await orderPage.inputICO.fill(ico);
+    await page.keyboard.press("Enter");
+
+    await page.waitForTimeout(5000);
+
+    await expect(orderPage.aresToast).toBeVisible();
```

Comment:
_page.waitForTimeout_ pokud možno nepoužívat. V tomto případě se dá nahradit následovně
```suggestion
    await expect(orderPage.aresToast).toBeVisible({ timeout: 5000 });
```